dashboard/config/linux/android-5.4.config

```diff
@@ -1331,7 +1341,54 @@ CONFIG_NET_FLOW_LIMIT=y
 # end of Networking options
 
 # CONFIG_HAMRADIO is not set
-# CONFIG_CAN is not set
+CONFIG_CAN=y
+CONFIG_CAN_RAW=y
+CONFIG_CAN_BCM=y
+CONFIG_CAN_GW=y
+# CONFIG_CAN_J1939 is not set
+
+#
+# CAN Device Drivers
+#
+# CONFIG_CAN_VCAN is not set
+# CONFIG_CAN_VXCAN is not set
```

Comment:
It can make sense to enable VCAN/VXCAN as well. I am not sure we have any of CAN devices otherwise and CAN stack won't be covered. But the virtual devices may to cover some of the CAN stack.

Reply:
This could be done as an explicit separate patch. The config was generated with syz-kconf, there's nothing I added manually. I'll take a look into VCAN/VXCAN, thanks for the tip.

Reply:
Right, let's keep it as vanilla as possible.  Subsequent config changes should be separate and obvious.

Not sure how relevant CAN devices are either.  We don't appear to have them enabled in ACK.